pkg/adapter/apiserver/config.go

```diff
@@ -32,6 +32,7 @@ type ResourceWatch struct {
 
 type Config struct {
 	// Namespaces specifies the namespaces where Resources[] exist.
+	// An empty slice means all namespaces
```

Comment:
what about a non-empty selector that matches 0 namespaces?

Reply:
ah whoops, good point. I'll update with a dedicated config option